test/checks/navigation/identical-links-same-purpose-after.js

```diff
@@ -0,0 +1,99 @@
+describe('identical-links-same-purpose-after tests', function() {
+	'use strict';
+
+	var fixture = document.getElementById('fixture');
+	var check = checks['identical-links-same-purpose'];
+
+	afterEach(function() {
+		fixture.innerHTML = '';
+	});
+
+	function assertResult(
```

Comment:
Abstracting in tests is not a good practice. Please just use the three asserts where you need to. The point of tests is that they should be easy to read. Tests don't need to be DRY.